### PR TITLE
event sampler: getBalance() is used to get total eth raised

### PR DESCRIPTION
**Pull request by pcppcp**

### CODE REVIEW

File: event_sampler/resources.py [hunk 1]
```diff
@@ -47,6 +48,7 @@ def get_histogram(self):
         max_block = max(block_to_events.keys())
         num_bins = args['bins']
 #        assert max_block > min_block
+#        import pudb;pudb.set_trace()
```
Comment (LefterisJP) [inline event_sampler/resources.py]: remove the comments here?

File: event_sampler/resources.py [hunk 2]
```diff
@@ -68,16 +70,20 @@ def get_histogram(self):
...
         web3 = self.sampler.chain.web3
         ret = {}
         ret['auction_stage'] = self.contract.call().stage()
         ret['price'] = self.contract.call().price()
         block_to_sum = {}
         for block, events in self.sampler.events.items():
             block_to_sum[block] = sum((event['args']['_amount'] for event in events))
+        total = sum((v for v in block_to_sum.values()))
+        wallet_balance = self.contract.web3.eth.getBalance(self.wallet_address)
+        if total != wallet_balance:
```
Comment (LefterisJP) [inline event_sampler/resources.py]: so if they don't match you simply just take the value of the wallet balance. Will this affect the graph? Also why not just take the wallet balance to begin with?

Comment (pcppcp) [reply event_sampler/resources.py]: that's just a check to raise a warning if for some reason stored events sum doesn't match real wallet balance. In the result returned from the endpoint actual balance is always used.

File: event_sampler/resources.py (hunk 1 above)
Comment (pcppcp) [reply event_sampler/resources.py]: sure.

**Pull request merged**